api/go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/rs/cors v1.7.0
 	github.com/stretchr/testify v1.6.1
-	github.com/tidwall/gjson v1.6.1
+	github.com/tidwall/gjson v1.6.5
```

Comment:
To fix Depandabot's security vulnerability alert:
<img width="938" alt="Screenshot 2021-10-07 at 01 47 47" src="https://user-images.githubusercontent.com/1886194/136294050-a98b8cd7-4d9a-44ad-8b3e-0ca53b4cf4df.png">